cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/service/AthenaServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.nike.cerberus.audit.logger.service;
+
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import com.nike.cerberus.audit.logger.AthenaClientFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+public class AthenaServiceTest {
+
+  AthenaClientFactory athenaClientFactory = new AthenaClientFactory();
+  @Mock private AthenaService mockAthenaService;
+  private AthenaService athenaService;
+
+  @Before
+  public void before() {
+    initMocks(this);
+    athenaService = new AthenaService("fake-bucket", athenaClientFactory);
+  }
+
+  @Test
+  public void test_that_addPartition_works() {
+    String fileName = "localhost-audit.2018-01-29_12-58.log.gz";
+    try {
+      athenaService.addPartitionIfMissing("us-west-2", "fake-bucket", "2018", "01", "29", "12");
+    } catch (Exception e) {
+      assertTrue(false);
```

Comment:
So is this assertion meant to make sure the method doesn't throw an Exception? You could just remove the try/catch. If it throws, then the exception will go up the stack and the test will fail. Otherwise you can use the fail() assertion instead of assertTrue(false).

Reply:
And this test doesn't actually test any logic inside that addPartitionIfMissing() method. How much value is this test providing if it doesn't test and assert that the function behaves as written?

Reply:
made changes